api/meetings/views.py

```diff
@@ -195,38 +186,6 @@ def annotate_queryset_with_author_name(self, queryset):
         )
         return queryset
 
-    def annotate_queryset_with_download_count(self, queryset):
-        """
-        Annotates queryset with download count of first osfstorage file
-
-        NOTE: This is a brittle way to do this.  PageCounter _ids are of the form
-        <file_action>:<node__id>:<file__id>:<sometimes version>.
-        - Assumes the "download" file action is the only action with that many letters
-        - Assumes node and file guids are a consistent length
-        - ENG-122 would get rid of this string matching behavior
-        """
-        pages = PageCounter.objects.annotate(
-            node_id=Substr('_id', 10, 5),
-            file_id=Substr('_id', 16),
-            _id_length=Length('_id'),
-        ).filter(
-            _id__icontains='download',
-            node_id=OuterRef('guids___id'),
-            file_id=OuterRef('file_id'),
-        ).exclude(_id_length__gt=39)
-
-        file_subqs = OsfStorageFile.objects.filter(
-            target_content_type_id=ContentType.objects.get_for_model(AbstractNode),
-            target_object_id=OuterRef('pk'),
-        ).order_by('created')
-
-        queryset = queryset.annotate(
-            file_id=Subquery(file_subqs.values('_id')[:1]),
-        ).annotate(
-            download_count=Coalesce(Subquery(pages.values('total')[:1]), Value(0)),
-        )
-        return queryset
```

Comment:
I propose _temporarily_ removing *sorting* functionality on download counts until [PageCounter optimizations are in](https://openscience.atlassian.net/browse/ENG-122), which is the purpose of this section. Because there is a sequence scan happening for PageCounters because PageCounters and Files are not actually related, this is too intensive a query right now.

Thanks to @brianjgeiger's excellent optimizations of the PageCounter work, I was able to populate my PageCounter table properly with the proposed columns that will accompany ENG-122. I verified that the sorting on download counts reached acceptable speeds with this fix.  Once PageCounter work is in, this method will be restored to look something like this:
```python
def annotate_queryset_with_download_count(self, queryset):
        """
        Annotates queryset with download count of first osfstorage file
        """
        pages = PageCounter.objects.filter(
            action='download',
            resource_id=OuterRef('guids__id'),
            file_id=OuterRef('file_id'),
            version=None
        )

        file_subqs = OsfStorageFile.objects.filter(
            target_content_type_id=ContentType.objects.get_for_model(AbstractNode),
            target_object_id=OuterRef('pk'),
        ).order_by('created')

        queryset = queryset.annotate(file_id=Subquery(file_subqs.values('id')[:1])).annotate(
            download_count=Coalesce(Subquery(pages.values('total')[:1]), Value(0)),
        )
        return queryset
```